channels/il.m3u

```diff
@@ -1,51 +1,37 @@
+https://reshet-live-il.ctedgecdn.net/13tv-premium/bb/r13_2400.m3u8
+#EXTINF:-1 tvg-id="Mikan-IS" tvg-name="כאן IL: Mikan" tvg-language="Arabic" tvg-logo="https://www.makan.org.il/images/logo_makan_33.png" group-title="General",ch 33 מכאן
+https://kanlivep2event-i.akamaihd.net/hls/live/747613/747613/source1_4k/chunklist.m3u8
+#EXTINF:-1 tvg-id="Knesset Channel-IS" tvg-name="Knesset Channel-IS" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/he/thumb/4/43/Knessetch10.svg/2560px-Knessetch10.svg.png" group-title="Legislative",ch 99 הכנסת
 http://w3.013.gostreaming.tv:1935/Knesset/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/lBbbPI4.jpg" group-title="Music",Music 24
-https://keshethlslive-lh.akamaihd.net/i/24live_1@195271/index_1400_av-b.m3u8
-#EXTINF:-1 tvg-id="Channel 13 Reshet-IS" tvg-name="IL: ISRAEL Channel 13 Reshet" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/WmgyRCF.png" group-title="",Reshet 13
-http://nana10-hdl-il-sw.ctedgecdn.net/Nana10-Live/amlst:hd_,500,1000,1200,2400,/playlist.m3u8
-#EXTINF:-1 tvg-id="Channel 13 Reshet-IS" tvg-name="IL: ISRAEL Channel 13 Reshet" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/WmgyRCF.png" group-title="",Reshet 13
-http://nana10-hdl-uk.ctedgecdn.net/Nana10-Live/amlst:hd_,500,1000,1200,2400,/playlist.m3u8
-#EXTINF:-1 tvg-id="Channel 13 Reshet-IS" tvg-name="IL: ISRAEL Channel 13 Reshet" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/WmgyRCF.png" group-title="",Reshet 13
-http://nana10-hdl.ctedgecdn.net/Nana10-Live/amlst:hd_%2C1000%2C1500%2C1800%2C/chunklist_b1000000.m3u8
-#EXTINF:-1 tvg-id="Shopping Channel-IS" tvg-name="IL: shopping channel" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/9AggY4H.png" group-title="Shop",Shopping Channel
-http://82.80.192.30/shoppingil_ShoppingIL21TVRepeat/_definst_/smil:ShoppingIL21TV.smil/chunklist_w2043660694_b2128000.m3u8?ttl=1524643697&cdn_token=5a6c4f1270440ae72118c8ae16c58de4
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/7JH43wB.jpg" group-title="",Yeshua Chai TV
-https://ihoplive-f.akamaihd.net/i/yeshuachai_tv@124973/index_2_av-p.m3u8?sd=10&rebase=on
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="" group-title="",Ynet Live
-https://ynet-lh.akamaihd.net/i/ynet_1@123290/index_1_av-p.m3u8?sd=10&rebase=on%7CUser-Agent=Mozilla/5.0%20(Windows%20NT%206.1;%20rv:40.0)%20Gecko/20100101%20Firefox/40.0
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="" group-title="",אתר ספורט 5
-https://sport5-lh.akamaihd.net/i/radiolivev_0@698733/index_1_av-p.m3u8?sd=10&rebase=on%7CUser-Agent=Mozilla/5.0%20(Windows%20NT%206.1;%20Win64;%20x64)%20AppleWebKit/537.36%20(KHTML,%20like%20Gecko)%20Chrome/44.0.2403.157%20Safari/537.36&Referer=http://radio.sport5.co.il
```

Comment:
This link works.

Reply:
Already on my list I committed. (note I changed the order, according to channel popularity more or less - the high rating channels first)

Reply:
Links in the playlist are automatically sorted alphabetically every time the playlist is updated.